mfa/methods/code/messaging.md

```diff
@@ -19,18 +19,18 @@ Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Cre
 
 | Parameter | Required | Description |
 |:--------------|:----------------------|:--|
-| to            | Yes | To telephone number   |
-| from          | Yes | From telephone number |
-| applicationId | Yes | The messaging application id |
+| to            | Yes | The phone number the message should be sent to (must be in E.164 format, like `+19195551212`). |
+| from          | Yes | One of your telephone numbers from which the message should come from (must be in E.164 format, like `+19195551212`). This number must be associated with the messaging application given by the `applicationId` parameter. |
+| applicationId | Yes | The messaging application ID. |
```

Comment:
maybe keep it camelcase if the parameter is camelcase, i.e. `The messaging application Id`

Reply:
I'll update but I don't think there's necessarily a right or wrong answer, and this is inconsistent across the dev site.